package.json

```
@@ -0,0 +1,44 @@
+{
+    "author": "Chiitoi",
```

This is a club project, so the author should be the club.
```suggestion
    "author": "Toronto Riichi Club",
```